### ISSUE TITLE
Move README content into Plone 6 docs, leaving only version information

### ISSUE THREAD

**Issue by stevepiercy** [plone/plone-backend#80]
READMEs should have only minimal information and not duplicate that which is already published in Plone 6 docs. This README currently has some non-duplicated information that should be moved into the docs, and replaced with a link to the docs.

See https://github.com/plone/plone-backend/pull/67#issuecomment-1280893500 for discussion, and suggested changes in https://github.com/plone/plone-backend/pull/67/files.

**Comment by jensens** [comment]
I think this was resolved already?

**Comment by stevepiercy** [comment]
Nope. Information was deleted, not moved. See https://github.com/plone/plone-backend/pull/67/files and the full readme at that commit https://github.com/Rudd-O/plone-backend/blob/ede17ae2cf1f3abccad90fee78b30b78effe51dd/README.md
